framework/model/module/module.go

```diff
@@ -424,19 +422,60 @@ func Main(bundle string, modules []Module) {
 			},
 			Stop: ctx.Done(),
 		}
+
+		mc.env = env
+		if err := mc.module.Init(ModuleOptions{
+			Env:               env,
+			InitCbs:           cbs,
+			Manager:           mgr,
+			LeaderElectionCbs: le,
+		}); err != nil {
+			log.Errorf("mod %s init met err %v", mc.env.Config.Name, err)
+			fatal()
+		}
+	}
+
+	// run ConfigController
```

Comment:
I'm afraid that the `configController` may emit events when it's not ready and those event-handler modules which depend on `configController` can not handle events correctly then.

Reply:
These eventhandlers mainly convert the cfg into cache